Sources/DatadogObjc/OpenTracing/OTSpan+objc.swift

```diff
@@ -7,7 +7,8 @@
 import Foundation
 
 /// Corresponds to: https://github.com/opentracing/opentracing-objc/blob/master/Pod/Classes/OTSpan.h
-@objc public protocol OTSpan {
+@objc
+public protocol OTSpan {
```

Comment:
Causing:
> Attributes Violation: Attributes should be on their own lines in functions and types, but on the same line as variables and imports. (attributes)

in swiftlint 0.50.1. I assume they fixed an issue and now it captures this case properly.